### PR TITLE
5.2

### PR DESCRIPTION
**Pull request by ana-dav**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
-
 public class Plane {
 
+    private String brand;
+    private int model;
+    private int power;
+    private String color;
+    private int wheelsDiameter;
+
+    public Plane(PlaneBuilder planeBuilder) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java:11]:
```suggestion
    private Plane(PlaneBuilder planeBuilder) {
```
explain why in comment below

Comment (ana-dav) [reply src/main/java/core/basesyntax/Plane.java:11]: This should be public, becarse this allows passing different types of builders to the director.

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -1,11 +1,64 @@
...
+        this.power = planeBuilder.power;
+        this.color = planeBuilder.color;
+        this.wheelsDiameter = planeBuilder.wheelsDiameter;
+    }
+
+    public static class PlaneBuilder {
+        //Required
+        private String brand;
+        private int model;
+        //Optional
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: remove comments

File: src/main/java/core/basesyntax/Plane.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/Plane.java:11]: I changed it to private) so please explain me  why it should be this way?

Comment (ana-dav) [reply src/main/java/core/basesyntax/Plane.java:11]: Oh, because we don't want to change fields of the Object throw Builder

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**